scripts/kong_backend/swap_ksusdt_to_sol.sh

```
@@ -0,0 +1,83 @@
+if [ -z "$1" ]
+	then
+		NETWORK=""
+	else
+		NETWORK="--network $1"
+fi
+IDENTITY="--identity kong_user1"	# cannot be mint account
+
+KONG_CANISTER=$(dfx canister id ${NETWORK} kong_backend)
+
+PAY_TOKEN="ksUSDT"
+PAY_TOKEN_LEDGER=$(dfx canister id ${NETWORK} $(echo ${PAY_TOKEN} | tr '[:upper:]' '[:lower:]')_ledger)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)